client/selectors/index.js

```diff
@@ -277,3 +277,12 @@ export const isCurrentPlanningLockedInThisSession = createSelector(
         currentPlanning.lock_session === session.sessionId ? true : false
     )
 )
+
+export const isEventDetailLockedInThisSession = createSelector(
+    [getShowEventDetails, getEvents, getSessionDetails],
+    (showEventDetails, events, session) => {
+        const event = events[showEventDetails]
+        return event && event.lock_user === session.identity._id &&
+        event.lock_session === session.sessionId ? true : false
```

Comment:
wow, this looks a bit mysterious if you forget the operator priorities. It would be a bit more explicit with parentheses.
Also I like to use `!!` to cast to boolean, but it's matter of taste.
Ex: `session.sessionId  ? true : false` === `!!session.sessionId`